cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -192,14 +197,19 @@ func (c *CStorPoolController) cStorPoolAddEventHandler(cStorPoolGot *apis.CStorP
 	}
 	var importPoolErr error
 	var status string
-	cachfileFlags := []bool{true, false}
-	for _, cachefileFlag := range cachfileFlags {
+	cachefileFlags := []bool{true, false}
+	for _, cachefileFlag := range cachefileFlags {
 		status, importPoolErr = c.importPool(cStorPoolGot, cachefileFlag)
 		if status == string(apis.CStorPoolStatusOnline) {
 			c.recorder.Event(cStorPoolGot, corev1.EventTypeNormal, string(common.SuccessImported), string(common.MessageResourceImported))
 			common.SyncResources.IsImported = true
 			return status, nil
 		}
+		if importPoolErr != nil {
+			glog.Errorf("Import succeded, but, other operations failed %v", importPoolErr)
```

Comment:
the log convention that we are trying to follow is as follows:
`<first phrase>: <2nd phrase>: ...`

- first phrase relates to what the method/function is trying to do
- [optional] 2nd & subsequent phrases are extra info that we may provide

#### sample
```go
glog.Errorf("failed to handle add event: import succeeded with failed operations: %v", importPoolErr)
```

_First phrase will be repeated across the method if this method has more than one log statements, however when seen from logs it helps to debug._